admin/components/RedirectForm/index.js

```diff
@@ -136,15 +136,17 @@ const RedirectForm = (props) => {
 export { RedirectForm };
 
 const FormSchema = (formatMessage) => {
-  const urlRegex = /^(?!www\.|(?:https?|ftp):\/\/|[A-Za-z]:\\|\/\/).+$/;
+  const relativeUrlRegEx = /^(?!www\.|(?:https?|ftp):\/\/|[A-Za-z]:\\|\/\/).+$/;
+  const absoluteUrlRegex = /^(www\.|(?:https?|ftp):\/\/|[A-Za-z]:\\|\/\/).+$/;

```

Comment:
This regex lets you also make:

```
www.https://something.com
```

I think you can omit the entire `www\.|` part and have a valid absolute URL matcher.